docs/telemetry.md

```diff
@@ -142,10 +141,23 @@ To elaborate on these events:
 | Event                                      | category | method         | object        | value          |
 |--------------------------------------------|----------|----------------|---------------|----------------|
 | PocketFeed: video click                    | action   | click          | video_id      | `<int>`        |
-| POcketFeed: video impression*              | action   | impression     | video_id      | `<int>`        |
+| PocketFeed: video impression*              | action   | impression     | video_id      | `<int>`        |
 
 (*) Video impressions are aggregated by the user initiating focus on the videos in the Pocket feed
 
+### Settings Channel
+
+| Event                                       | category   | method      | object        | value          |
+|---------------------------------------------|------------|-------------|---------------|----------------|
+| Settings tile clicked*                      | action     | click       | setting       |                |
+| Confirms clear data dialog                  | action     | change      | setting       | clear_data     |
+| Clear cookies and data tile clicked         | action     | click       | setting       | clear_data_tile|
+| Send usage data tile clicked                | action     | click       | setting       | send_data_tile |
+| About tile clicked                          | action     | click       | setting       | about_tile     |
+| Privacy Notice tile clicked                 | action     | click       | setting       | privacy_tile   |
+
+(*) This event is the total number of clicks on all Settings tile. The other events breakdown which tile was clicked.
```

Comment:
Since we can sum these up, I think that we don't need to collect the total number of settings tiles clicked.

Reply:
Ok, I will remove the total count.